packages/camera/android/src/main/java/io/flutter/camera/CameraPlugin.java

```diff
@@ -198,35 +205,30 @@ public void onMethodCall(MethodCall call, Result result) {
         }
       case "start":
         {
-          long textureId = ((Number) call.argument("textureId")).longValue();
-          Cam cam = cams.get(textureId);
+          Cam cam = getCamOfCall(call);
           cam.start();
           result.success(true);
```

Comment:
We usually do `result.success(null)` if there is nothing to communicate except that the call completed.

Here and other places below.

Reply:
Done